config.yml

```
@@ -1915,7 +1915,11 @@ nodes:
           ^^^^^^^^^^^^^^
   - name: MultiTargetNode
     fields:
-      - name: targets
+      - name: requireds
```

```suggestion
      - name: pres
```
I would suggest `pres` or `pre` here (and for MultiWriteNode), to match with `post/posts`.
Both of them are not really required in the context of multiple assignment, they are just the things to assign before and after the `rest`.

Although I guess `requireds` was chosen to match https://github.com/ruby/prism/blob/87a9deb6bd8d8251d58977d394ab947c5600495d/config.yml#L2022-L2032 but the concept of required doesn't really exist for multiple assignment (everything is optional).

---

Renamed to `lefts`/`rights`